src/amp.rs

```diff
@@ -60,10 +84,16 @@ impl Amp {
         Ok(Amp { sender })
     }
 
-    fn get_pin() -> Result<OutputPin, Error> {
-        let mut pin: OutputPin = Gpio::new()?.get(Amp::AMP_SD_GPIO_PIN)?.into_output();
-        pin.set_low();
-        Ok(pin)
+    fn get_pin_power() -> Result<OutputPin, Error> {
+        let mut pin_power: OutputPin = Gpio::new()?.get(Amp::AMP_POWER_GPIO_PIN)?.into_output();
```

Comment:
See comment above: i would not like this thing to explode if we run it on a laptop. We could alternatively also use compile time configurations to handle this use case, though!

Reply:
I see, alright. Moving the error handling into `get_pin_x` wouldn't cut it then because we need a defined result. Have to get the hang on rusty thinking

Reply:
I know this looks ugly (and implementation wise, i am sure it can be simplified), but this is what it's all about: it is safe, the behavior is well defined :)